app/views/moderations/mod.html.erb

```diff
@@ -245,15 +104,21 @@
         <button value="9" name="rating_vote[rating]" class="level-rating-button <%= "selected" if @rating_vote&.rating == 9.0 %>">9</button>
         <button value="10" name="rating_vote[rating]" class="level-rating-button <%= "selected" if @rating_vote&.rating == 10.0 %>">10</button>
       <% end %>
-      <p style="width: 90%;margin:50px auto;text-align:left">
-        <b style="font-size:1.1em">All rating votes are private.</b>
-        <br><br>Provide a rating between 1 and 10 based on which audience would find this post most valuable.
-        <br><br>1: for brand new developers, 3: for junior developers, 10: for very advanced developers, etc.
-        <br><br>This will provide
-        <em>approximate and gentle</em> algorithmic adjustments to help deliver relevant content.
-        <br><br>It will be used as one of many indicators. No one rating will have an overly dramatic affect.
+      <p>
+        <br><br>Who will find this post most valuable, based on experience level?
```

Comment:
The line break tag `<br>` is [considered deprecated and a potential frustration for screen readers[(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br). Do you think we could get rid of it and use multple paragraph tags instead?

Reply:
Great call. I typically think of BR tags as a total hack to accomplish things, but this is further reason to get them out.

Reply:
I think for short term stuff they can be fine, especially in places like internal, but they should only be used for prototyping imo. Even then I try not to use them.